go/fuzz/pgp_key_read/fuzz.go

```diff
@@ -8,7 +8,7 @@ package libkb
 import "github.com/keybase/client/go/libkb"
 
 func Fuzz(data []byte) int {
-	key, err := libkb.ReadOneKeyFromBytes(data)
+	key, err := libkb.ReadOneKeyFromBytes(nil, data)
```

Comment:
My preference would be to define a wrapper method (possibly with the original name) that doesn't take a context, and instead passes in `libkb.G`, rather than having the contextified version explicitly handle nil. Curious what @maxtaco thinks though.